src/schematools/types.py

```diff
@@ -351,13 +352,10 @@ def _expand_relation_spec(
             to_snake_case(part) for part in str(relation).split(":")[:2]
         ]
 
-        # XXX maybe not logical to snakecase the fieldname here.
-        # this is still schema-land.
-        snakecased_fieldname = to_snake_case(field.name)
         snakecased_field_id = to_snake_case(field.id)
         table_id = get_rel_table_identifier(len(self.id) + 1, table.id, snakecased_field_id)
 
-        sub_table_schema: Json = {
+        sub_table_schema: Dict[str, Any] = {
```

Comment:
Why this change. Surely the alias `Json` communicates the type of dictionary better then merely `Dict[str, Any]`?

Reply:
https://github.com/Amsterdam/schema-tools/blob/75ad36b29d3f0b5c3e97d86c9233ad97546f5440/src/schematools/types.py#L39

Of these, only `Dict[str, Any]` supports the indexing by string below.